.github/workflows/release.yml

```diff
@@ -1,6 +1,4 @@
 name: Draft release
-run-name: Draft release ${{ inputs.next_version }}
```

Comment:
Why removing that? I find it handy to know which version was tagged on the Workflows overview page, without having to go see the logs.

Reply:
again you are right, putting it back